src/living/helpers/gui2d/control.ts

```diff
@@ -739,9 +759,14 @@ export class Control2D {
       element.width = rect.width;
       element.height = rect.height;
     });
-    renderingContext.putImageData(this._imageData, rect.x, rect.y, 0, 0, rect.width, rect.height);
+    renderingContext.drawImage(this._imageData, rect.x, rect.y, rect.width, rect.height);
   }
-
+  
+  _updateTransform(renderingContext: CanvasRenderingContext2D = this._renderingContext) {
```

Comment:
No need to pass an argument of `renderingContext`, because it always be `this._renderingContext`, feel free to correct me if I'm wrong :)